network-api/networkapi/templates/pages/base.html

```diff
@@ -22,7 +22,7 @@
     {% endblock %}
 
     {% block ga_identifier %}
-      <meta name="google-site-verification" content="Hl4u7xOmnhODNRyNoTjSzBFJKizAYXWYtzxfHjVkPRc">
+      <meta name="google-site-verification" content="D7k-r3fHm-XfJ9E7T1uZ5aqHJG2mx-0uUZFeBUDN2lY" />
```

Comment:
```suggestion
      <meta name="google-site-verification" content="D7k-r3fHm-XfJ9E7T1uZ5aqHJG2mx-0uUZFeBUDN2lY">
```
No closing tag on `<meta>` in HTML5 (if this were JSX, then yes)

Reply:
interesting, that code was autogenerated by the Google Search Console site, so they're adding invalid HTML to pages all across the web!